META.in

```diff
@@ -2,7 +2,7 @@ version = """
 PiGx SARS-CoV-2 wastewater sequencing Pipeline.
 Version: @PACKAGE_VERSION@
 
-Copyright © 2021 Akalin Lab
+Copyright © 2022 Akalin Lab
```

Comment:
Leave the old year there as well.  Should be 2021, 2022.

Reply:
Fixed now.